File: src/main/java/com/campool/model/BookingInfo.java
package com.campool.model;

import java.time.LocalDate;
import lombok.Getter;
import lombok.RequiredArgsConstructor;
import lombok.ToString;

@Getter
@RequiredArgsConstructor
@ToString
public class BookingInfo {

    private final long bookingId;

    private final LocalDate startDate;

    private final LocalDate endDate;

    private final int amount;

    private final long rentalId;

    private final String title;

    private final String description;

    private final String rentalStatus;

    private final double longitude;

    private final double latitude;

    private final String userId;

    private final String name;

    private final String email;

    private final String telephone;

}

File: src/main/java/com/campool/model/BookingInfo.kt
package com.campool.model

import java.time.LocalDate
import com.campool.enumeration.RentalStatus

data class BookingInfo(
    val bookingId: Long,
    val startDate: LocalDate,
    val endDate: LocalDate,
    val amount: Int,
    val rentalId: Long,
    val title: String,
    val description: String,
    val rentalStatus: RentalStatus,
    val longitude: Double,
    val latitude: Double,
    val userId: String,
    val name: String,
    val email: String,
    val telephone: String
)